engine/src/main/resources/templates/ImageBuilder/launchFailedEmailBodyV2.html

```diff
@@ -0,0 +1,29 @@
+{@boolean renderSection1=true}
+{@boolean renderSection1AsIncident=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    Image launch - Red Hat Enterprise Linux
+{/content-title}
+{#content-title-section1}
+    An image failed to launch on
+{#switch action.context.provider}
+{#case "aws"}
+Amazon Web Services
+{#case "azure"}
+Microsoft Azure
+{#case "gcp"}
+Google Cloud Platform
+{/switch}
+{/content-title-section1}
+{#content-subtitle-section1}
+{action.timestamp.toStringFormat()} | launch id #{action.context.launch_id}
```

Comment:
Timestamp should be used only if different than email timestamp, is that the case ? 
If yes we can keep it here.

Reply:
emails could be delayed, no? isn't the timestamp more accurate when the event occurred.